include/gatekeeper_config.h

```diff
@@ -67,11 +73,38 @@ struct dynamic_config {
 
 	/* The Lua file for initializing dynamic configuration. */
 	char             *dynamic_config_file;
+
+	struct mailbox   mb;
+	/*
+	 * The callee that finished processing the return message
+	 * needs to increment this counter, so that the Dynamic config block
+	 * can finish its operation.
+	 */
+	rte_atomic16_t   num_returned_instances;
+};
+
+/* Define the possible command operations for Dynamic config block. */
+enum dy_cmd_op {
+	GT_UPDATE_POLICY_RETURN,
+};
+
+/* Currently, the GT blocks are the only writers of Dynamic config mailbox. */
+struct dy_cmd_entry {
+	enum dy_cmd_op  op;
+
+	union {
+		/* GT_UPDATE_POLICY_RETURN */
+		struct {
+			unsigned int gt_lcore;
+			char return_msg[RETURN_MSG_MAX_LEN];
+		} gt;
+	} u;
 };
 
 int config_gatekeeper(const char *lua_base_dir,
 	const char *gatekeeper_config_file);
 int set_lua_path(lua_State *l, const char *path);
+int l_update_gt_lua_states_incrementally(lua_State *l);
```

Comment:
`l_update_gt_lua_states_incrementally()` is only refered in file `config/dynamic.c` where it's defined, so we can drop this prototype.